File: Assignment_5/Question5.c
#include<stdio.h>

int CheckLargest(int iNo1, int iNo2, int iNo3)
{
    if(iNo1 > iNo2 && iNo1 > iNo3)
    {
        return iNo1;
    }
    else if(iNo2 > iNo1 && iNo2 > iNo3)
    {
        return iNo2;
    }
    else
    {
        return iNo3;
    }
}

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;
    int iValue3 = 0;

    int iResult = 0;

    printf("Enter the first number : \n");
    scanf("%d", &iValue1);

    printf("Enter the second number : \n");
    scanf("%d", &iValue2);

    printf("Enter the third number : \n");
    scanf("%d", &iValue3);

    iResult = CheckLargest(iValue1, iValue2, iValue3);

    printf("The largest number is: %d\n", iResult);

    return 0;
}

File: Assignment__19/Question5.java
//Write a program to calculate the power of a number using loops.

import java.util.Scanner;

class Function
{
    int CalculatePower(int iNo1, int iNo2)
    {
        int iCnt = 0;
        int iPower = 1;

        for(iCnt = 1; iCnt <= iNo2; iCnt++)
        {
            iPower = iPower * iNo1;
        }

        return iPower;
    }
}

class Question5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number: ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter the Power: ");
        int iValue2 = sobj.nextInt();

        int iResult = 0;
        

        Function fobj = new Function();

        iResult = fobj.CalculatePower(iValue1, iValue2);

        System.out.println("The Answer is : " + iResult);

    }

}
